### DISCUSSION TITLE
multiple selections for categories, tags and tools

### DISCUSSION THREAD

**Discussion by CamelT0E** [mealie-recipes/mealie#1377] [Feature Request]
great job and a nice to have

I have 2 suggestions for improvement:

please add multiple selections support for categories, tags and tools.
otherwise, you always have to open each recipe to assign it to the above criteria.

The language setting could also be set in the setting.

**Comment by michael-genson** [comment]
I think what you're asking for is in the recipe data management page:
![image](https://github.com/user-attachments/assets/45faabdc-b0ae-4849-a78a-35bafa6c7800)

If not, if you could elaborate and re-open that would be great
